packages/migrate/tsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "../../tsconfig.json"
+  "extends": "../../tsconfig.json",
+  "include": ["./**/*"]
```

Comment:
This ensures that only the files at the root and the project are included, making it clear that we don't need to look below. I think this could also help with some eslint slowdowns but not 100% sure. This should be mostly harmless anyways.